ERROR - 2022-01-11 05:39:55 --> Error connecting to the database: No such file or directory
CRITICAL - 2022-01-11 05:39:55 --> Unable to connect to the database.
Main connection [MySQLi]: No such file or directory
#0 /app/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /app/app/Models/UserModel.php(18): CodeIgniter\Database\BaseConnection->query('SELECT * FROM u...')
#2 /app/app/Controllers/Home.php(17): App\Models\UserModel->get_users()
#3 /app/system/CodeIgniter.php(824): App\Controllers\Home->jeanpaul()
#4 /app/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 /app/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /app/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2022-01-11 05:40:23 --> Error connecting to the database: No such file or directory
CRITICAL - 2022-01-11 05:40:23 --> Unable to connect to the database.
Main connection [MySQLi]: No such file or directory
#0 /app/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /app/app/Models/UserModel.php(18): CodeIgniter\Database\BaseConnection->query('SELECT * FROM u...')
#2 /app/app/Controllers/Home.php(17): App\Models\UserModel->get_users()
#3 /app/system/CodeIgniter.php(824): App\Controllers\Home->jeanpaul()
#4 /app/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 /app/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /app/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2022-01-11 05:41:13 --> Error connecting to the database: No such file or directory
CRITICAL - 2022-01-11 05:41:13 --> Unable to connect to the database.
Main connection [MySQLi]: No such file or directory
#0 /app/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /app/app/Models/UserModel.php(18): CodeIgniter\Database\BaseConnection->query('SELECT * FROM u...')
#2 /app/app/Controllers/Home.php(17): App\Models\UserModel->get_users()
#3 /app/system/CodeIgniter.php(824): App\Controllers\Home->jeanpaul()
#4 /app/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 /app/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /app/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2022-01-11 05:58:46 --> Error connecting to the database: Access denied for user '****'@'78-72-143-119-no190.tbcn.telia.com' (using password: YES)
CRITICAL - 2022-01-11 05:58:46 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'78-72-143-119-no190.tbcn.telia.com' (using password: YES)
#0 /app/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /app/app/Models/UserModel.php(18): CodeIgniter\Database\BaseConnection->query('SELECT * FROM u...')
#2 /app/app/Controllers/Home.php(17): App\Models\UserModel->get_users()
#3 /app/system/CodeIgniter.php(824): App\Controllers\Home->jeanpaul()
#4 /app/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 /app/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /app/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
